azure/petctl.py

```diff
@@ -0,0 +1,348 @@
+from util import *
+import argparse
+
+# Create a Kubernetes specs and YAML job file based on user inputs
+def configure(args):
+    configure_yaml(args)
+    configure_json(args)
+
+
+# Deploys a Kubernetes cluster
+def setup(args):
+    # Login to Azure
+    azure_login()
+    # Install AKS Engine
+    install_aks_engine()
+    # Deploy an AKS cluster using kubernetes.json
+    deploy_aks_cluster(args)
+
+
+# Upload code/data to Azure blob storage
+def upload_storage(args):
+    upload_to_azure_blob(args)
+
+
+# Create Azure blob storage secret
+def storage_secret(args):
+    create_storage_secrets(args)
+
+
+# Create docker image secrets
+def docker_secret(args):
+    create_docker_image_secret(args)
+
+
+# Submits your training job
+def run_job(args):
+    install_blobfuse_drivers()
+    commands = [
+        "kubectl delete -f config/azure-pytorch-elastic.yaml",
+        "kubectl apply -f config/azure-pytorch-elastic.yaml",
+        "kubectl describe pods",
+        "kubectl get pods --selector app=azure-pytorch-elastic",
+    ]
+
+    run_commands(commands)
+
+
+# Check current status of your pods
+def check_status():
+    commands = [
+        "kubectl describe pods",
+        "kubectl get pods --selector app=azure-pytorch-elastic",
+    ]
+
+    run_commands(commands)
+
+
+# Get logs of your job from each pod
+def get_logs():
+    run_commands(["kubectl logs --selector app=azure-pytorch-elastic "])
+
+
+# Deletes secrets and cluster
+def delete_resources():
+    commands = [
+        "kubectl config delete-cluster azure-pytorch-elastic",
+        "kubectl delete secret pet-blob-secret",
+        "kubectl delete namespace --all",
+        "RD /S /Q _output",
```

Comment:
This only works on Windows?

Reply:
Changed to shutl.rmtree, thanks for pointing this out!